sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/data/redis/lettuce/AzureRedisCredentials.java

```diff
@@ -25,9 +28,11 @@ public class AzureRedisCredentials implements RedisCredentials {
      * Create instance of Azure Redis Credentials
      */
     public AzureRedisCredentials(String username, PasswordlessProperties passwordlessProperties) {
-        Objects.requireNonNull(passwordlessProperties, "PasswordlessProperties is required");
+        Objects.requireNonNull(passwordlessProperties, "PasswordlessProperties is required.");
         azureAuthenticationTemplate = new AzureAuthenticationTemplate();
-        azureAuthenticationTemplate.init(passwordlessProperties.toPasswordlessProperties());
+        Properties properties = passwordlessProperties.toPasswordlessProperties();
+        enhancePasswordlessProperties("spring.data.redis.azure", passwordlessProperties, properties);
```

Comment:
What's this prefix for?

Reply:
It's a debug logging format for passwordless.